wandb/sdk/launch/docker.py

```diff
@@ -225,14 +230,23 @@ def build_docker_image_if_needed(
             f"ENV WANDB_PROJECT={launch_project.target_project}",
             f"ENV WANDB_ENTITY={launch_project.target_entity}",
             f"ENV WANDB_LAUNCH={True}",
-            f"ENV WANDB_LAUNCH_CONFIG_PATH={os.path.join(workdir,_project_spec.DEFAULT_LAUNCH_METADATA_PATH)}",
+            f"ENV WANDB_LAUNCH_CONFIG_PATH={os.path.join(workdir, DEFAULT_LAUNCH_METADATA_PATH)}",
             f"ENV WANDB_RUN_ID={launch_project.run_id or None}",
             f"ENV WANDB_DOCKER={launch_project.docker_image}",
         ]
     )
     dockerfile_contents += env_vars + "\n"
 
-    launch_project._dockerfile_contents = dockerfile_contents
+    sanitized_dockerfile_contents = re.sub(
+        r"WANDB_API_KEY=\w+", "WANDB_API_KEY", dockerfile_contents
```

Comment:
nit: can you make a single regex constant for this substitution and use it for both

Reply:
Fixed